dna/data.py

```diff
@@ -160,17 +160,42 @@ def encode_dag(dag: typing.Sequence[typing.Sequence[typing.Any]]):
     """
     return ''.join(''.join(str(edge) for edge in vertex) for vertex in dag)
 
-
-def preprocess_data(train_data, test_data):
+def filter_metafeatures(instance: dict, metafeature_subset: str):
+    metafeatures = instance['metafeatures']
```

Comment:
Let's keep this function general and pass the metafeatures, not the whole instance dict into this function.